mqtt/mqtt-bridge/src/pump.rs

```diff
@@ -267,10 +271,8 @@ impl Pump {
                         debug!("extracted publication from store");
 
                         if let Ok(Some((key, publication))) = loaded_element {
-                            debug!("publishing {:?}", key);
-                            if let Err(e) = publish_handle.publish(publication).await {
-                                error!(err = %e, "failed publish");
-                            }
+                            debug!("publishing publication {:?}", key);
+                            publish_handle.publish(publication).await;
```

Comment:
We now log an error internally. Mqtt3 will retry itself right? If it fails this retry limit and errors then I feel the correct approach is to not have our own retry logic here, but just log the error.